### PR TITLE
Store last read location on device

### PR DESCRIPTION
**Pull request by sebastiandve**

### CODE REVIEW

File: client/app.json [hunk 1]
```diff
@@ -13,24 +13,35 @@
...
     "experiments": {
       "typedRoutes": true
+    },
+    "extra": {
+      "router": {
+        "origin": false
+      },
+      "eas": {
+        "projectId": "6f69be41-4d10-43a7-9777-c3b5b3c12d6a"
+      }
```
Comment (sebastiandve) [inline client/app.json:44]: not sure if we can delete this

File: client/app.json [hunk 2]
```diff
@@ -13,24 +13,35 @@
...
+      "supportsTablet": true,
+      "bundleIdentifier": "com.anonymous.src"
     },
     "android": {
       "adaptiveIcon": {
         "foregroundImage": "./assets/images/adaptive-icon.png",
         "backgroundColor": "#ffffff"
-      }
+      },
+      "package": "com.anonymous.src"
```
Comment (reginaczech) [inline client/app.json:24]: do our packages need to be the same? I've done com.magicink

File: client/app.json (hunk 1 above)
Comment (reginaczech) [reply client/app.json:44]: I think a project id is required to create the url for the build, so i wouldnt delete this yet

**Review by reginaczech — DISMISSED** [review]

**Comment by sebastiandve** [comment]
DO NOT MERGE

**Pull request closed without merging**